networkit/cpp/distance/test/DistanceGTest.cpp

```diff
@@ -611,4 +612,78 @@ TEST_P(DistanceGTest, testPrunedLandmarkLabeling) {
     });
 }
 
+TEST_P(DistanceGTest, testDynPrunedLandmarkLabelingAddEdge) {
+    /**	 0       3
+     *	  \     / \
+     *	   1---2   4
+     */
+
+    Graph G(5, isWeighted(), isDirected());
+
+    G.addEdge(0, 1);
+    G.addEdge(2, 3);
+    G.addEdge(3, 4);
+
+    DynPrunedLandmarkLabeling pll(G);
+    pll.run();
+
+    EXPECT_EQ(pll.query(0, 1), 1);
+    EXPECT_EQ(pll.query(2, 3), 1);
+    EXPECT_EQ(pll.query(2, 4), 2);
+
+    EXPECT_EQ(pll.query(0, 2), none);
+    EXPECT_EQ(pll.query(1, 2), none);
+
+    G.addEdge(1, 2);
+    pll.update(GraphEvent(GraphEvent::EDGE_ADDITION, 1, 2, 1));
+
+    EXPECT_EQ(pll.query(0, 1), 1);
+    EXPECT_EQ(pll.query(2, 3), 1);
+    EXPECT_EQ(pll.query(2, 4), 2);
+
+    EXPECT_EQ(pll.query(0, 2), 2);
+    EXPECT_EQ(pll.query(1, 2), 1);
+
+    EXPECT_EQ(pll.query(0, 3), 3);
+    EXPECT_EQ(pll.query(1, 3), 2);
+
+    EXPECT_EQ(pll.query(0, 4), 4);
+    EXPECT_EQ(pll.query(1, 4), 3);
+}
+
+TEST_P(DistanceGTest, testDynPrunedLandmarkLabelingBuildClique) {
```

Comment:
Really minor: If we look at the coverage, both the [header](https://github.com/networkit/networkit/actions/runs/4460282338/jobs/7833317305#step:6:175) and the [implementation](https://github.com/networkit/networkit/actions/runs/4460282338/jobs/7833317305#step:6:554) are not at 100%. I think we are missing tests for illegal `GraphEvents` and `updateBatch`.

Reply:
Done.